stix_shifter_modules/aws_cloud_watch_logs/stix_translation/json/to_stix_map.json

```diff
@@ -598,46 +598,45 @@
         }
       ],
       "detail_resource_instanceDetails_imageId":{
-        "key": "x_aws_instance.image_id",
-        "cybox": false
+        "key": "x-aws-instance.image_id",
+        "object": "x_aws_instance"
       },
       "detail_resource_instanceDetails_instanceId": {
-        "key": "x_aws_instance.instance_id",
-        "cybox": false
+        "key": "x-aws-instance.instance_id",
+        "object": "x_aws_instance"
       },
       "detail_resource_instanceDetails_availabilityZone": {
-        "key": "x_aws_instance.availability_zone",
-        "cybox": false
+        "key": "x-aws-instance.availability_zone",
+        "object": "x_aws_instance"
       }
     },
     "@timestamp": {
-        "key": "x_aws_guardduty_finding.timestamp",
-        "cybox": false,
-        "transformer": "AwsToTimestamp"
+        "key": "x-aws-guardduty-finding.timestamp",
+        "transformer": "AwsToTimestamp",
+        "object": "x_aws_guardduty_finding"
     },
     "account": {
-      "key": "x_aws.account_id",
-      "cybox": false
+      "key": "x-aws.account_id",
+      "object": "x_aws"
     },
     "detail_region": {
-      "key": "x_aws.region",
-      "cybox": false
+      "key": "x-aws.region",
+      "object": "x_aws"
     },
     "detail_id": {
-      "key": "x_aws_guardduty_finding.id",
-      "cybox": false
+      "key": "x-aws-guardduty-finding.id",
```

Comment:
Just make this map to `x-aws.detail_id` and the object `x_aws`, like how `detail_region` is getting mapped above.